src/intel-map.coffee

```diff
@@ -10,8 +10,6 @@
 # Commands:
 #   hubot intelmap for <search>
 #
-# Author:
-#   therealklanni
```

Comment:
:frowning: 

If you want credit as a contributor, you can submit a PR for the README, but please don't remove the author's name from the script.